artview/components/limits.py

```diff
@@ -207,6 +207,6 @@ def limits_dialog(limits, cmap, name):
         limits['xmax'] = float(ent_xmax.text())
         limits['ymin'] = float(ent_ymin.text())
         limits['ymax'] = float(ent_ymax.text())
-        print limits
+        print(limits)
```

Comment:
Oh man, still finding these code issues!
Question: Is this print statement needed?